homeassistant/components/frontend/__init__.py

```diff
@@ -533,6 +533,7 @@ async def resolve(
         """
         if (
             request.path != "/"
+            and len(request.url.parts) > 1
```

Comment:
No longer need to check path.

```suggestion
            len(request.url.parts) > 1
```

Reply:
I had originally done that, but two tests failed:
```
FAILED tests/components/frontend/test_init.py::test_frontend_and_static[pyloop] - AssertionError: assert 404 == 200
FAILED tests/components/frontend/test_init.py::test_we_cannot_POST_to_root[pyloop] - AssertionError: assert 404 == 405
```
Those failures seemed to me like a potentially breaking change in behavior, so I re-added the path check.

Reply:
We want to get this check only if the url is not `/`, e.g. if it is `/map`. Removing the `request.path != "/"` will enter this also for root url, since the url parts reported are `['/','']`